doc/help/configuring.asciidoc

```diff
@@ -120,7 +120,7 @@ c.completion.shrink = True
 ----
 
 Note that qutebrowser does some Python magic so it's able to warn you about
-mistyped config settings. As an example, if you do `c.tabs.possition = "left"`,
+mistyped config settings. As an example, if you do `c.tabs.position = "left"`,
```

Comment:
This is intentional, see the text above

Reply:
God, I'm so dumb for not noticing that (I even read this exact documentation before!). I'll fix it in a minute.